nomos-services/cryptarchia-consensus/src/lib.rs

```diff
@@ -48,21 +48,22 @@ pub enum Error {
     Consensus(#[from] cryptarchia_engine::Error<HeaderId>),
 }
 
-struct Cryptarchia {
-    ledger: cryptarchia_ledger::Ledger<HeaderId>,
-    consensus: cryptarchia_engine::Cryptarchia<HeaderId>,
+#[derive(Debug, Clone)]
```

Comment:
because [`proptest::StateMachineTest`](https://docs.rs/proptest-state-machine/latest/proptest_state_machine/test_runner/trait.StateMachineTest.html) requires these.